src/frontends/common/include/openvino/frontend/graph_iterator.hpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/runtime_attribute.hpp"
+
+namespace ov {
+namespace frontend {
+
+class IGraphIterator : RuntimeAttribute {
+public:
+    virtual ~IGraphIterator() = default;
+};
```

Comment:
In any case now you introduce new class `GraphIterator` in the common frontend. So let us move existing `GraphIterator` base class to the common frontend and create binding for it

Reply:
Done.